Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift

```diff
@@ -1126,7 +1132,9 @@ class VisualRecognitionTests: XCTestCase {
                 } else {
                     // verify the image's custom classifier
                     XCTAssertEqual(classifier.classifierID, self.classifierID!)
-//                    XCTAssertEqual(classifier.name, self.classifierName)
+                    if classifier.classifierID != classifier.name {
```

Comment:
Right now Visual Recognition actually returns the classifier name for the classifier ID, so I added this check until the Visual Recognition team either fixes it, or changes the information returned.